migrations/0011_add_properties_to_order.up.sql

```diff
@@ -0,0 +1,6 @@
+alter table order_items
+add location text,
```

Comment:
Do we need location on each order item?  I was under the impression from this PR that the order will also have the same location as all the order items.

Reply:
That's a good point, it's easier on the client side C++ if we keep it like this though.

Reply:
:+1: